internal/model/plan.go

```diff
@@ -0,0 +1,79 @@
+package model
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+// Plan defines the structure for an Api Plan.
+// swagger:model
+type Plan struct {
+	//The id for the plan
+	// in: path
+	//required: true
+	ID *string `gorm:"type:uuid;default:uuid_generate_v1()" json:"id"`
+	// The Name for the plan
+	//required: true
+	Name string `gorm:"not null;unique" json:"name"`
+	// The Description for the plan
+	//required: true
+	Description string `gorm:"not null" json:"description"`
+	// PlanQuotaDefaults []PlanQuotaDefaults `json:"quota_defaults"`
+
+}
+
+// PlanQuotaDefaults define the structure for an Api Plan and Quota.
+// swagger:model
+type PlanQuotaDefaults struct {
```

Comment:
If you rename this model to `PlanQuotaDefault` then GORM will automatically name the table `plan_quota_defaults`. That will also allow you to remove the `TableName` method.